src/materials/NEMLThermalExpansionEigenstrain.C

```diff
@@ -40,20 +40,17 @@ NEMLThermalExpansionEigenstrain::NEMLThermalExpansionEigenstrain(const InputPara
   _model = neml::parse_xml_unique(_fname, _mname);
 }
 
-void
-NEMLThermalExpansionEigenstrain::computeThermalStrain(Real & thermal_strain,
-                                                      Real * instantaneous_cte)
+ValueAndDerivative<false>
+NEMLThermalExpansionEigenstrain::computeThermalStrain()
 {
-  double nemlCTE = _model->alpha(_temperature[_qp]);
+  double nemlCTE = _model->alpha(MetaPhysicL::raw_value(_temperature[_qp]));
   double nemlCTE_old = _model->alpha(_temperature_old[_qp]);
 
-  thermal_strain =
+  const auto thermal_strain =
       _tstrain_old[_qp] + (nemlCTE + nemlCTE_old) / 2 * (_temperature[_qp] - _temperature_old[_qp]);
+  _tstrain[_qp] = MetaPhysicL::raw_value(thermal_strain);
```

Comment:
...another instance of the _thermal_strain we were just talking about!

Reply:
Yeah, I'll update that with the new `dual_number_cast`

Reply:
Isn't T-strain a rapper?